public/.htaccess

```diff
@@ -1,20 +1,21 @@
 <IfModule mod_rewrite.c>
     <IfModule mod_negotiation.c>
-        Options -MultiViews
+        Options -MultiViews -Indexes
```

Comment:
@keithnet shouting out the addition of the `-Indexes` option here. I think it's fine, but I don't know enough to have strong opinions about .htaccess files. Let me know if you would like to see something else.

Reply:
This was simply added because it is part of Laravel's default install today 👀

Reply:
Thank you for clarifying!